Source/Details/ASPINRemoteImageDownloader.mm

```diff
@@ -231,15 +231,15 @@ - (void)cachedImageWithURL:(NSURL *)URL
 {
   [[self sharedPINRemoteImageManager] imageFromCacheWithURL:URL processorKey:nil options:PINRemoteImageManagerDownloadOptionsSkipDecode completion:^(PINRemoteImageManagerResult * _Nonnull result) {
     [ASPINRemoteImageDownloader _performWithCallbackQueue:callbackQueue work:^{
-      const BOOL isFromMemoryCache = result.resultType == PINRemoteImageResultTypeMemoryCache;
+      ASImageCacheType source = (result.resultType == PINRemoteImageResultTypeMemoryCache ? ASImageCacheTypeSynchronous : ASImageCacheTypeAsynchronous);
```

Comment:
nit: I'd probably call this `cacheType` because that's how we've named it elsewhere.

Reply:
Fixed, thanks.